src/core/hooks/useHideScrollbar.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect } from 'react';
+
+const useHideScrollbar = (active: boolean, onlyMobile = false): void => {
+  useEffect(() => {
+    const isActive = (): boolean => {
+      if (onlyMobile) return window.innerWidth < 768;
+      else return true;
+    };
+    if (isActive()) {
+      const scrollbarWidth = window.innerWidth - document.body.clientWidth;
+      console.log(scrollbarWidth);
+      if (active) {
+        document.body.style.overflow = 'hidden';
+        if (scrollbarWidth) document.body.style.paddingRight = `${scrollbarWidth}px`;
+      } else {
+        document.body.style.overflow = 'unset';
+        document.body.style.paddingRight = '0';
+      }
+    }
+  }, [active, onlyMobile]);
+};
+
+export default useHideScrollbar;
```

Comment:
Just for my information: can you explain shortly what causes this issue and how it is resolved?

Reply:
Let me explain after daily